atom/browser/api/atom_api_session.cc

```diff
@@ -503,11 +492,13 @@ v8::Local<v8::Promise> Session::ClearAuthCache() {
 }
 
 void Session::AllowNTLMCredentialsForDomains(const std::string& domains) {
-  base::PostTaskWithTraits(
-      FROM_HERE, {BrowserThread::IO},
-      base::BindOnce(&AllowNTLMCredentialsForDomainsInIO,
-                     WrapRefCounted(browser_context_->GetRequestContext()),
-                     domains));
+  auto* command_line = base::CommandLine::ForCurrentProcess();
+
+  auto auth_params = ::network::mojom::HttpAuthDynamicParams::New();
+  auth_params->server_whitelist = domains;
+  auth_params->delegate_whitelist = command_line->GetSwitchValueASCII(
+      atom::switches::kAuthNegotiateDelegateWhitelist);
```

Comment:
Do we also want to preserve the value of `enable_negotiate_port` here ? Since the default value from the service is not what we want.

Reply:
great catch, yes.